spec/collection-spec.coffee

```diff
@@ -1,18 +1,33 @@
-describe 'Brainstem.Collection', ->
-  collection = updateArray = null
+$ = require 'jquery'
+_ = require 'underscore'
+Backbone = require 'backbone'
```

Comment:
I can try and move this config to the gulp file but I'm not sure the module support so passing in the config.